nbdime/webapp/nbdimeserver.py

```diff
@@ -239,6 +240,7 @@ def make_app(**params):
     settings = {
         "static_path": static_path,
         "template_path": template_path,
+        "exit_code": 0,
```

Comment:
I would maybe not put this in `settings`, as `tornado` uses that object. Put it as an attribute?